tests/Postman/Mage2 Framework.postman_collection.json

```diff
@@ -0,0 +1,430 @@
+			"item": [
+				{
+					"name": "check_cart",
+					"description": null,
+					"item": [
+						{
+							"name": "Check_cart android",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "a6dcc1f0-89d1-44aa-9655-4fc29bffdf61",
+										"type": "text/javascript",
+										"exec": [
+											"pm.test(\"Cart has coupon\", function () {",
+											"    var jsonData = pm.response.json();",
+											"    pm.expect(jsonData.external_coupons).not.to.be.empty;",
+											"});"
+										]
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "*/*"
+									},
+									{
+										"key": "X-Shopgate-Auth-User",
+										"value": "{{sgHeaderAuthUser}}"
+									},
+									{
+										"key": "X-Shopgate-Auth-Token",
+										"value": "{{sgHeaderAuthToken}}"
+									}
+								],
+								"body": {
+									"mode": "formdata",
+									"formdata": [
+										{
+											"key": "action",
+											"value": "check_cart",
+											"type": "text"
+										},
+										{
+											"key": "shop_number",
+											"value": "{{shop_number}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][item_id]",
+											"value": "{{simple_item_id}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][item_number]",
+											"value": "{{simple_item_number}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][item_number_public]",
+											"value": "{{simple_item_number_public}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][internal_order_info]",
+											"value": "{{simple_item_internal_order_info}}",
+											"description": "",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][quantity]",
+											"value": "1",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][currency]",
+											"value": "USD",
+											"type": "text"
+										},
+										{
+											"key": "cart[currency]",
+											"value": "USD",
+											"type": "text"
+										},
+										{
+											"key": "cart[client][type]",
+											"value": "androidphoneapp",
+											"description": "",
+											"type": "text"
+										}
+									]
+								},
+								"url": {
+									"raw": "{{domain}}",
+									"host": [
+										"{{domain}}"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Check_cart iphone",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "10626eaf-08dc-436d-af3c-888c077eb2aa",
+										"type": "text/javascript",
+										"exec": [
+											"pm.test(\"Cart has coupon\", function () {",
```

Comment:
how does the coupon exist in the cart? does it need to be created manually? If so, this will not do. The test needs to create the coupon and import it into magento. What happens is that all tests do not need any configuration in magento except the shopgate credential stuff.

Reply:
I see.
But I don't see how I would create a cart rule in magento, except for extending our API and creating a function for that. Which is totally out of scope

Reply:
See add_order cron tests. We create an order, then prompt mage to Ship the order via Mage API, then run cron, then check database values via mage api again.

Reply:
doesn't work that way. added sql file, which adds a coupon